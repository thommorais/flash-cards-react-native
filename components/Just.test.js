import React from 'react'
import renderer, { create } from 'react-test-renderer'
import Jest from './Jest'

test('teste',  () => {
    const tree = renderer.create(<Jest />).toJSON()
    expect(tree).toMatchSnapshot()
})