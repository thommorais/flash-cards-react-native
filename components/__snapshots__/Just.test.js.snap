// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#EBEDF0",
        "borderRadius": 20,
        "flex": 1,
        "padding": 2,
      },
      Object {
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderRadius": 20,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "paddingBottom": 2,
          "paddingTop": 2,
        },
        Object {
          "backgroundColor": "#F6D548",
        },
        Object {
          "width": "33.33333333333333%",
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontWeight": "bold",
          },
          Object {
            "color": "purple",
          },
        ]
      }
    >
      33
      %
    </Text>
  </View>
</View>
`;
